blocks/form/form.js

```diff
@@ -0,0 +1,85 @@
+async function fetchData(url) {
+  const resp = await fetch(url);
+  const json = await resp.json();
+  return json.data;
+}
+
+function onChange(form) {
+  // Targeting the anchor inside .button-container
+  const submitButton = form.querySelector('input[type="submit"]');
+  const emailInput = form.querySelector('input[type="email"]');
+
+  const allCheckboxesChecked = [...form.querySelectorAll('input[type="checkbox"]')].every((checkbox) => checkbox.checked);
```

Comment:
Could you set the second checkbox as not required as it is also not required in the original page?
<img width="660" alt="Screenshot 2023-08-23 at 17 18 39" src="https://github.com/hlxsites/bitdefender/assets/30388999/b8e1ebf0-b4d1-4af8-9240-349ba1f7b1db">